src/App.js

```diff
@@ -1,27 +0,0 @@
-import logo from './logo.svg';
```

Comment:
- [ ] App.js is not present in your application. This milestone requires App.js and it should be a class-based component. Kindly endeavor to improve upon this aspect